### ISSUE TITLE
Clusters style

### ISSUE THREAD

**Issue by camillemonchicourt** [makinacorpus/Geotrek-rando#507]
People that have been trying Geotrek-rando V2 around, didnt understand clusters as they are displayed actually, they think it is a content with a number : 
![clusters-v2](https://cloud.githubusercontent.com/assets/4418840/12475603/518dc12a-c024-11e5-92fd-8ad63c052840.jpg)

They had no problem with the previous style with the object picto, a different border colour and a number when it is a cluster. 
![clusters-v1](https://cloud.githubusercontent.com/assets/4418840/12475632/88d3b2b6-c024-11e5-8f78-8ccdbf6f4d30.jpg)

I know that this style can't work as now clusters can include different kind of objects but maybe there is a good idea to make clusters easier to understand for users.

**Comment by gutard** [comment]
In v1 we cluster only objects of the same type (trek), with a single pictogram.
in v2 we cluster objects of differents types, with different pictograms.
What pictogram do you expect when we cluster a trek together with a restaurant ?
Note: clustering objects by type (pictogram) independantly is not a solution (potentially too many clusters, and cluster may overlap)

**Comment by camillemonchicourt** [comment]
Yes indeed, that's what I explained and as you say, clustering on each category would not be a good solution.
